docs/getting_started.rst

```diff
@@ -106,50 +34,51 @@ will not work.
    `cd /path/to/virtualenv'
    source bin/activate or . bin/activate
 
-* Install python dependencies:
-
-  * ``PYCURL_SSL_LIBRARY=nss pip install -Ur /path/to/virtualenv/cfme_tests/requirements.txt``
-  * If you get error from pycurl and you used this command, you might like to remove pycurl and try
```

Comment:
The pycurl-related info is valuable as well; we had people run into this issue multiple times before.